autoload/test/python/djangotest.vim

```diff
@@ -39,6 +39,8 @@ function! test#python#djangotest#executable() abort
     let pipenv_prefix = "pipenv run "
   elseif filereadable("poetry.lock")
     let pipenv_prefix = "poetry run "
+  elseif filereadable("uv.lock")
+    let pipenv_prefix = "uv run "
```

Comment:
If this doesn't work with django I'd remove this

```suggestion
```

Reply:
I guess I'll do that